checks/cloud/aws/redshift/no_classic_resources.cf.go

```diff
@@ -2,23 +2,11 @@ package redshift
 
 var cloudFormationNoClassicResourcesGoodExamples = []string{
 	`---
-AWSTemplateFormatVersion: 2010-09-09
-Description: Good example of redshift sgr
-Resources:
-
```

Comment:
was this intentional?

Reply:
Yes. This check is related to resources using EC2-Classic, so the example provided is not relevant. Additionally, [AWS has completely retired EC2-Classic](https://aws.amazon.com/blogs/aws/ec2-classic-is-retiring-heres-how-to-prepare/) as of August 23, 2023, so this check should probably be retired altogether.

Reply:
I understand. Your point about deprecating the check makes sense but I do think we should do that in a separate PR once we have added support for deprecating a check and for each such deprecated check we can update the docs separately. I've opened [this](https://github.com/aquasecurity/trivy/issues/6626) to track adding support for deprecation of checks. 

For the time being I would bring back these examples and as I mentioned we can open a separate PR to update both check and its docs. 

Thoughts?

Reply:
Sure, I will restore these to their previous state.

Reply:
Thanks @StevenSmiley - looks like the tests are red. If you could update them we can merge this PR in. Thanks for the patience.

Reply:
I have updated the tests

Reply:
Still seem to be failing. You can run `make test` locally to run the tests.